src/main/java/com/odysseusinc/arachne/portal/api/v1/controller/BaseAdminController.java

```diff
@@ -142,10 +154,52 @@ public void fullDeleteStudy(Long[] ids) {
             UserSearch userSearch)
             throws UserNotFoundException {
 
-        Page<IUser> users = userService.getAll(pageable, userSearch);
+        Page<U> users = (Page<U>) userService.getAll(pageable, userSearch);
```

Comment:
The main goal of adding generics is to eliminate explicit typecastings, so pls use BaseUserService w/ generics